src/main.cpp

```diff
@@ -2455,8 +2455,11 @@ bool FindBlockPos(CValidationState &state, CDiskBlockPos &pos, unsigned int nAdd
     }
 
     nLastBlockFile = nFile;
-    vinfoBlockFile[nFile].nSize += nAddSize;
     vinfoBlockFile[nFile].AddBlock(nHeight, nTime);
+    if (fKnown && vinfoBlockFile[nFile].nSize < (pos.nPos + nAddSize))
+        vinfoBlockFile[nFile].nSize = pos.nPos + nAddSize;
```

Comment:
When can I fetch latest source code, compile and test?

On Mon, Mar 9, 2015 at 1:01 PM, Jeff Garzik <notifications@github.com>
wrote:

> In src/main.cpp
> <https://github.com/bitcoin/bitcoin/pull/5864#discussion_r26029884>:
>
> >      vinfoBlockFile[nFile].AddBlock(nHeight, nTime);
> > +    if (fKnown && vinfoBlockFile[nFile].nSize < (pos.nPos + nAddSize))
> > +        vinfoBlockFile[nFile].nSize = pos.nPos + nAddSize;
>
> +1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/5864/files#r26029884>.
>



-- 
Mvh
-fredrik-normann-

Sent from my Gmail Account